CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Coding guidelines
```

Comment:
I've taken your feedback on board and added the ``Bug Reports`` category. I couldn't find a need for the rest given the context of someone reading that file when they're filing an issue or submitting a PR - let me know if you'd like to see any specific categories.

In general, I find that file to be overwhelming for a first-time contributor who just wants to submit a 1 line fix and you have to remember that the scale of Rust and Mudlet are totally different, so will be the people who contribute to it (hardcode C++ experts vs people who learnt coding copy/pasting code in Mudlet).